### ISSUE TITLE
Need to accept terms before use

### ISSUE THREAD

**Issue by marcusbooyah** [nforgeio/neonKUBE#1673]
Azure.RequestFailedException: 'User failed validation to purchase resources. Error message: 'You have not accepted the legal terms on this subscription: 'af47a13f-fd84-4989-b969-e268d0e5d2c3' for this plan. Before the subscription can be used, you need to accept the legal terms of the image. To read and accept legal terms, use the Azure CLI commands described at https://go.microsoft.com/fwlink/?linkid=2110637 or the PowerShell commands available at https://go.microsoft.com/fwlink/?linkid=862451. Alternatively, deploying via the Azure portal provides a UI experience for reading and accepting the legal terms. Offer details: publisher='neonforgellc1660679895021' offer = 'neonkube_0-8-0-preview', sku = 'neonkube', Correlation Id: '8ebbdb34-3a93-4c42-b61d-08a959a4d6ab'.'

We might want to see if we can do this in neon desktop

**Comment by jefflill** [comment]
I don't believe that's possible (I looked into that a while ago).  The cloud providers need the user to actually visit the marketplace page and agree there.  Perhaps we can detect whether the user has agreed or not and send them to the product agreement page.

**Comment by jefflill** [comment]
Now that AWS Marketplace images exist, we probably need to do the same for AWS.

**Comment by jefflill** [comment]
For Azure, it looks like it's possible to accept terms using PowerShell cmdlets:

[set-azurermmarketplaceterms](https://learn.microsoft.com/en-us/powershell/module/azurerm.marketplaceordering/set-azurermmarketplaceterms?view=azurermps-6.13.0)
[set-azmarketplaceterms](https://learn.microsoft.com/en-us/powershell/module/az.marketplaceordering/set-azmarketplaceterms?view=azps-3.4.0)
